src/models/document/document-content.ts

```diff
@@ -133,6 +133,9 @@ export const DocumentContentModel = types
       getTileType(tileId: string) {
         return self.tileMap.get(tileId)?.content.type;
       },
+      getTileTitle(tileId: string) {
+        return self.tileMap.get(tileId)?.title ?? "<no title>";
```

Comment:
I wonder if it would make more sense to just return `self.tileMap.get(tileId)?.title` here, and make a separate function like `getTileTitleForLogging` to switch it to `"<no title>"` when there is no title? It seems strange to me to return a string like that from a generic sounding function.

Reply:
Agreed -- the `<no title>` default is part of the logging, not part of the `DocumentContent`.